### ISSUE TITLE
Improve contrast of Badge and Activity Badge

### ISSUE THREAD

**Issue by suliskh** [khutwah/web#113]
### Description
Our current color combination of `Badge` and `ActivityBadge` components does not meet Web Content Accessibility Guidelines (WCAG) 2.1 contrast requirements, which can make our application difficult to read for users with visual impairments.

| Variant | Contrast |
|--------|--------|
|  <img width="64" alt="image" src="https://github.com/user-attachments/assets/d49e43fa-e78f-4c6a-9367-d0a4b56edf8a"> (green) | 3.67 |
| <img width="64" alt="image" src="https://github.com/user-attachments/assets/f4098017-a57d-482c-8862-3646182e5594"> (tamarind) | 1.89 |
| <img width="64" alt="image" src="https://github.com/user-attachments/assets/acf52a2b-7ee2-4e81-b9c9-1a3fc7c3b8c5"> (mute) | 1.68 | 

### Accessibility Standards

Target WCAG 2.1 Level AA contrast ratios:
- Normal text: 4.5:1 minimum
- Large text (14pt and bold or 18pt): 3:1 minimum
- User interface components: 3:1 minimum

### Proposed Actions
- Update color combination based on https://github.com/khutwah/khutwah-web/pull/108
